lib/perl/Genome/Model/ClinSeq/Command/CreateMutationSpectrum.pm

```diff
@@ -59,6 +59,11 @@ class Genome::Model::ClinSeq::Command::CreateMutationSpectrum {
             default => 0,
         },
     ],
+    has_param => [
+        lsf_resource => {
+            default => "-R 'select[tmp>1000] span[hosts=1] rusage[tmp=1000]' -M 16000000",
```

Comment:
The rusage should use gtmp (MGI inhouse resource] instead of tmp, and is indicated in Gb. This would reserve 1 Gb of tmp space:
```
rusage[gtmp=1]
```